Parsers/ASimNetworkSession/Parsers/vimNetworkSessionCheckPointFirewall.yaml

```diff
@@ -253,14 +253,16 @@ ParserQuery: |
                             "No match"
                         )
         | where ASimMatchingIpAddr != "No match"
+        | lookup ActionLookup on DeviceAction
+        | where ((array_length(dvcaction) == 0) or DvcAction has_any (dvcaction))
```

Comment:
The test for EventResult also has to come here instead of lines 243-245 given success is not just "accept".